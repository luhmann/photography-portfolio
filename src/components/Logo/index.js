import styled from 'styled-components';
import { color, space } from 'styled-system';

const Logo = styled.h1`
  ${color};
  ${space};
`;

Logo.propTypes = {
  ...color.propTypes,
  ...space.propTypes,
};

Logo.displayName = 'Logo';

export default Logo;
