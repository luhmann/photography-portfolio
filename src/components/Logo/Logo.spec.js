import React from 'react';
import { render } from 'react-testing-library';

import Logo from './';

test('should render as expected', () => {
  const { container } = render(<Logo />);

  expect(container.firstChild).toMatchSnapshot();
});

test('should contain the logo text', () => {
  const { getByText } = render(<Logo />);

  expect(getByText('JF Dietrich Photography')).toBeInTheDocument();
});
