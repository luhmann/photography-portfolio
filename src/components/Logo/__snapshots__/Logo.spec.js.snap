// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render as expected 1`] = `
<h1
  class="sc-bdVaJa NctWQ"
>
  JF Dietrich Photography
</h1>
`;
